doc/sphinx/using/libraries/funind.rst

```diff
@@ -1,6 +1,15 @@
 Functional induction
 ====================
 
+.. note::
+
+   Most features of the ``FunInd`` plugin are also available in the
+   ``Equations`` plugin, which is provided in the Coq Platform. For
+   new projects / new code, it is recommended using ``Equations`` instead
+   of ``FunInd``. ``FunInd`` is not deprecated and not planned for removal
+   yet because porting code from ``FunInd`` to ``Equations`` is difficult
+   (incompatibilities in the generated induction principles).
```

Comment:
Can we do a little better on this explanation?

```suggestion
   We recommend using the ``Equations`` plugin for new projects and new code
   instead of ``FunInd`` because <EXPLAIN WHY>.
   Most features of the ``FunInd`` plugin are also available in the
   ``Equations`` plugin, which is provided in the Coq Platform.
   ``FunInd`` is not deprecated and not planned for removal
   yet because porting code from ``FunInd`` to ``Equations`` is difficult
   (incompatibilities in the generated induction principles).
```

Equations is the replacement for FunInd?  Could say that more explicitly.
When would you still use FunInd for new code (what features does it have that Equations does not)?

Less important: Is it relevant to users whether something is a plugin or a library?  Is there any important difference?  If not, I would go with "library" throughout the documentation.

Reply:
Equations subsumes Function in almost every aspect: it supports dependent types whereas Function does not. It supports all ways of defining well founded recursive functions that Function supports. The only thing aspect I am aware of where it does not subsume Function is that the functional induction principles generated by Equations are less user-friendly.

Reply:
So yes Equation is a replacement for Function IIUC.

Reply:
> Less important: Is it relevant to users whether something is a plugin or a library? Is there any important difference? If not, I would go with "library" throughout the documentation.

I don't know. I tend to make the distinction. What does @palmskog think?

Reply:
I think the distinction between "library" and "plugin" has become less clear over time but is still valuable, for three key reasons:
- nearly all plugins add something to the vernacular language that could not be added otherwise (modulo MetaCoq), so users can know by the word "plugin" that they get new top-level commands or tactics.
- plugins can do almost anything to Coq, so they can't be fully trusted - when you add a plugin (as opposed to a library) this is a useful indicator that that the trusted base may increase, which could be mitigated by `coqchk` or similar
- ML plugins that are not in Coq's CI will bitrot quickly, which users likely will want to know before they spend time to adopt in their project

Reply:
I pushed a new version.